### PR TITLE
removed reward flicker fix

### PR DESCRIPTION
**Pull request by mobiusTripper-crypto**
Deployed a fix on the backend:
https://github.com/beethovenxfi/beethovenx-backend/pull/139
https://github.com/beethovenxfi/beethovenx-backend/pull/138

that appears to have solved the "flicker" problem. and also solved the problems with displaying the rewards in the navbar. so this PR removes the frontend code that was used as a work around.

Found secondary problem, after claiming rewards the reward token balance is showing NaN in the navbar.
image

This is cause by a bug in numeral.js, which is quite out of date, so I replaced that instance with `numbro' a fork of numeral.js that seems to be maintained. We can make a new task to swap those two libraries in the future.

### CODE REVIEW

**Comment by danielmkm** [comment]
just use the token formatters from `lib/services/token/token-util.ts` for the time being. Happy for you to swap out numeral, but just don't do it in a one off spot. Do it all at once. The formatters in token-util take account for the NAN

File: package.json [hunk 1]
```diff
@@ -66,5 +66,8 @@
         "typescript": "4.6.4",
         "use-debounce": "^8.0.1",
         "wagmi": "^0.5.12"
+    },
+    "dependencies": {
+        "numbro": "^2.3.6"
```
Comment (danielmkm) [inline package.json:71]: when you do add dependencies, make sure you add them as dev dependencies, reference all of the other dependencies in this file.

File: lib/global/useStakingPendingRewards.ts [hunk 2]
```diff
@@ -50,26 +49,12 @@ export function useStakingPendingRewards(stakingItems: GqlPoolStaking[]) {
...
-                }
-
-                pendingRewards = [...pendingRewards, ...currentGaugePendingRewards.current];
+                pendingRewards = [...pendingRewards, ...gaugePendingRewards];
             }
 
             return pendingRewards;
         },
-        { enabled: !!userAddress && stakingItems.length > 0, refetchInterval: 15000 },
+        { enabled: !!userAddress && stakingItems.length > 0, refetchInterval: 1000 },
```
Comment (danielmkm) [inline lib/global/useStakingPendingRewards.ts:57]: Why did you change the refetch interval to every second here?

**Comment by danielmkm** [comment]
Not entirely sure how this would fix the flicker on the pool detail, since it doesn't use this query from the backend anyway? It looks like you've changed the refetchInterval to 1 second, which is way to quick IMO

File: lib/global/useStakingPendingRewards.ts (hunk 2 above)
Comment (mobiusTripper-crypto) [reply lib/global/useStakingPendingRewards.ts:57]: This was only for testing, thanks for seeing that

File: package.json (hunk 1 above)
Comment (mobiusTripper-crypto) [reply package.json]: noted

**Review by danielmkm — APPROVED** [review]
